mantidproject/mantid
# Use CalMuonDetectorPhases in frequency domain analysis GUI

The frequency domain analysis GUI currently (v 3.11) uses a rough calculation for the phase table. This replaces the rough calculation with the algorithm CalMuonDetectorPhases. This allows for a more robust creation of the phase table. 

The phase table is used when combining the frequency spectra. 

**To test:**
1. Open Muon analysis and select MUSR as the instrument.
2. Enter 62260 as the run number (some data should then load) 
3. Expand the group workspace MUSR00062260
4. Close muon analysis
5. Open the frequency domain analysis GUI (in muon)
6. Change the workspace to phaseQuad
7. Change the first good data to 0.5
8. press the calculate FFT button. A phaseQuad workspace will appear in MUSR00062260
9. Open the phaseTable and change one of the phases (you will have to make it non-read only)
10. Untick the construct phase table option
11. Press the calculate FFT button
12. Check the phaseTable - your change should  still be there
13. Tick the construct phase table option
14. press the calculate FFT button 
15. Open the phaseTable and it will have reverted. 



Fixes #21419. 

**Release Notes** 

In the release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
